src/css/inputs.css

```diff
@@ -126,4 +126,23 @@
     .text-input-errorFree {
         @apply border-theme-primary-700 dark:border-theme-primary-650  focus:border-theme-primary-700 focus:dark:border-theme-primary-650 disabled:bg-theme-secondary-100
     }
+
+    /** Passphrase Input **/
+    .passphrase-primary, 
+    .passphrase-destructive,
+    .passphrase-errorFree {
+        @apply text-light-black dark:text-white border border-solid bg-white dark:bg-subtle-black focus:border focus:border-solid
+    }
+    
+    .passphrase-primary {
+        @apply border-theme-secondary-400 focus:border-light-black dark:focus:border-theme-secondary-300 disabled:bg-theme-secondary-100 read-only:bg-white dark:read-only:bg-subtle-black
+    }
+
+    .passphrase-destructive {
+        @apply border-theme-error-300 focus:border-theme-error-600 dark:focus:border-theme-error-500
+    }
+
+    .passphrase-errorFree {
+        @apply border-theme-primary-700 dark:border-theme-primary-650 focus:border-theme-primary-700 dark:focus:border-theme-primary-650 disabled:bg-theme-secondary-100
+    }
```

Comment:
will add a card to refactor this to be generic classes, as they are all the same for inputs (textarea, passphrase, etc) so this only causes duplication